andela/healthchecks-invincibles
# #150798554 Alerts via SMS and Telegram

#### What does this PR do?
This PR contains features to allow a user be notified via multiple means which are SMS and Telegram about their jobs.
#### Description of Task to be completed?
The user is given the option to add a reference phone number or provide Telegram username that will be used for notifications. When the phone number and Telegram username have been provided, the user is able to receive notifications
#### How should this be manually tested?

- Clone the repository and git checkout into this branch with `git checkout ft-alert-on-multiple-150798554` and install the requirements with `pip install -r requirements.txt`. 

- Create an account on Twilio. Get your Twilio number, Account SID and API token. Add them to your environment variables. 

- Create an account on Telegram. Interact with the botFather to create a bot. For further instructions check [here](https://manybot.io/). Add the API token to the the environment variables. 

- Run the application and see results

#### Any background context you want to provide?
Currently, alerts on a user's job are sent via email. With this update, users will be able to get notifications via Telegram and SMS too. 
#### What are the relevant pivotal tracker stories?
#150798554
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/11851029/30705019-fd31de22-9efc-11e7-94e9-4e87a3cfd770.png)

#### Questions:
